cpp/data/system_updates/6.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -o errexit -o nounset
+
+readonly SYSTEMD_SERVICE_DIR=/etc/systemd/system/
+mkdir --parents $(SYSTEMD_SERVICE_DIR)
+install --owner=root --group=root --mode=644 \
+        /usr/local/ub_tools/cpp/data/installer/boot_notification.service \
+        $(SYSTEMD_SERVICE_DIR)
+/usr/bin/systemctl start boot_notification.service
```

Comment:
When systemd service files are changed, you need to execute `systemctl daemon-reload` before accessing the service to make sure that the daemon has detected the new file

Reply:
Also, i guess you do not only want to start it, but also to enable it permanently. This might be even more important because if you start it, the mail will be sent after installation; more important is that we want to send it after each reboot.

Reply:
Also => do we really also want this to be installed on our development machines?